database_schemas/cassandra/benchflow.cql

```diff
@@ -182,6 +186,109 @@ CREATE TABLE host_properties (
   PRIMARY KEY(host_id)
 );
 
+-- Faban
+CREATE TABLE faban_run_info (
+  host text,
+  name text,
+  version text,
+  run_id text,
+  start_time text,
+  end_time text,
+  metric_unit text,
+  metric_value double,
+  passed boolean,
+  experiment_id text,
+  trial_id text,
+  PRIMARY KEY ((trial_id, experiment_id), host)
+);
+
+CREATE TABLE faban_driver_summary (
+  host text,
+  name text,
+  metric_unit text,
+  metric_value double,
+  start_time text,
+  end_time text,
+  total_ops_unit text,
+  total_ops_value double,
+  users bigint,
+  rtXtps double,
+  passed boolean,
+  experiment_id text,
+  trial_id text,
+  PRIMARY KEY ((trial_id, experiment_id), host)
+);
+
+CREATE TABLE faban_driver_mix (
+  host text,
+  driver_name text,
+  op_name text,
+  allowed_deviation double,
+  successes bigint,
+  failures bigint,
+  mix double,
+  required_mix double,
+  passed boolean,
+  experiment_id text,
+  trial_id text,
+  PRIMARY KEY ((trial_id, experiment_id), host, driver_name, op_name)
+);
+
+CREATE TABLE faban_driver_response_times (
+  host text,
+  driver_name text,
+  op_name text,
+  unit text,
```

Comment:
You are repeating this information each line, but it is defined once per `responseTimes`